src/providers/AppState.js

```diff
@@ -53,6 +54,7 @@ function AppStateProvider({ children }) {
         accountBalance: balance,
         convictionVoting,
         effectiveSupply,
+        errorFetchingOrg,
```

Comment:
And here pass errors instead.
```suggestion
       errors,
```